libs/auth/src/mock_file_card.ts

```diff
@@ -19,7 +19,6 @@ const MOCK_FILE_PATH = '/tmp/mock-file-card.json';
 export interface MockFileContents {
   cardStatus: CardStatus;
   data?: Buffer;
-  numIncorrectPinAttempts?: number;
```

Comment:
Realized that I could just use the existing `numIncorrectPinAttempts` field within `CardStatus` (`cardStatus.cardDetails.numIncorrectPinAttempts`) instead of creating a new field